lms/views/lti/config.py

```diff
@@ -2,7 +2,7 @@
 
 
 @view_config(
-    route_name="lti.v11.config",
+    route_name="lti.v11.canvas.config",
```

Comment:
The v11 endpoint it's also canvas specific so using the same name conventions in both.